sdk/js/index.ts

```diff
@@ -52,9 +63,9 @@ export class Stream {
     publicClient: PublicClient,
     walletClient?: WalletClient
   ): Promise<Stream> {
-    const creator = log.topics[2] as Address;
-    const streamId = Number(log.topics[3]);
-    const token = log.topics[4] as Address;
+    const creator = ("0x" + (log.topics[2] as string).slice(-40)) as Address;
+    const streamId = isNaN(Number(log.topics[3])) ? 1 : Number(log.topics[3]);
```

Comment:
when it's NaN, I think it should raise

Reply:
With the change from fetchLogs to getcontractevents (using eventName: "StreamCreated") we dont get streamID issues, as the logs are already cleaned